modules/ob-creating-a-s2i-build.adoc

```diff
@@ -43,6 +43,12 @@ spec:
     pushSecret: registry-credential <5>
 EOF
 ----
+<1> The location where the source code is placed.
+<2> The build strategy that you use to build the container.
+<3> The parameter defined in the build strategy. To set the value of the `builder-image` strategy parameter, specify the builder image location required to build the output image.
+<4> The location where the built image is pushed. You can push the built image to a custom Quay repo. Replace `repo` with a valid Quay organization or your Quay user name.
```

Comment:
I typically wouldn't hold up a merge for this, but there's a few other things I'm commenting on below.

Here, though, I suggest the following:

```suggestion
<4> The location where the built image is pushed. You can push the built image to a custom Quay.io repository. Replace `repo` with a valid Quay.io organization or your Quay user name.
```

The reason I'm suggesting this is because the examples are using Quay.io (eg --builder-image="quay.io/centos7/nodejs-12-centos7") and not "Red Hat Quay". They're essentially the same product offered in two different ways. But for here, we'd probably be best saying "Quay.io repository." (fwiw my main product is Quay). When we say "Quay" it's probably clear enough, but *could* mean Red Hat Quay OR Quay.io.